### PR TITLE
HBASE-22277 Removed deprecated methods from Get

### PR DESCRIPTION
**Pull request by HorizonNet**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 39 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 17 new or modified test files. |
||| _ master Compile Tests _ |
| 0 | mvndep | 29 | Maven dependency ordering for branch |
| +1 | mvninstall | 272 | master passed |
| +1 | compile | 190 | master passed |
| +1 | checkstyle | 202 | master passed |
| +1 | shadedjars | 271 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 485 | master passed |
| +1 | javadoc | 146 | master passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 17 | Maven dependency ordering for patch |
| +1 | mvninstall | 321 | the patch passed |
| +1 | compile | 222 | the patch passed |
| +1 | javac | 222 | the patch passed |
| +1 | checkstyle | 225 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 332 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 737 | Patch does not cause any errors with Hadoop 2.7.4 or 3.0.0. |
| +1 | findbugs | 608 | the patch passed |
| +1 | javadoc | 168 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 205 | hbase-client in the patch passed. |
| -1 | unit | 14420 | hbase-server in the patch failed. |
| +1 | unit | 1376 | hbase-mapreduce in the patch passed. |
| +1 | unit | 324 | hbase-thrift in the patch passed. |
| +1 | unit | 1012 | hbase-backup in the patch passed. |
| +1 | unit | 446 | hbase-rest in the patch passed. |
| +1 | asflicense | 191 | The patch does not generate ASF License warnings. |
| | | 22424 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/173 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 631339c42012 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 226c90c058 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/1/testReport/ |
| Max. process+thread count | 4549 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server hbase-mapreduce hbase-thrift hbase-backup hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-client/src/main/java/org/apache/hadoop/hbase/client/Get.java [hunk 1]
```diff
@@ -260,30 +228,6 @@ public Get setTimestamp(long timestamp) {
...
-  public Get setMaxVersions() {
-    return readAllVersions();
-  }
-
-  /**
-   * Get up to the specified number of versions of each column.
-   * @param maxVersions maximum versions for each column
-   * @throws IOException if invalid number of versions
-   * @return this for invocation chaining
-   * @deprecated It is easy to misunderstand with column family's max versions, so use
```
Comment (saintstack) [inline hbase-client/src/main/java/org/apache/hadoop/hbase/client/Get.java]: For sure deprecated since 2.0.0?

Comment (HorizonNet) [reply hbase-client/src/main/java/org/apache/hadoop/hbase/client/Get.java]: Deprecated with HBASE-17125, which has a fix version set to 2.0.0.

**Comment by HorizonNet** [comment]
Retriggered the build. Waiting for a +1 from it.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 77 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 17 new or modified test files. |
||| _ master Compile Tests _ |
| 0 | mvndep | 43 | Maven dependency ordering for branch |
| +1 | mvninstall | 298 | master passed |
| +1 | compile | 188 | master passed |
| +1 | checkstyle | 197 | master passed |
| +1 | shadedjars | 273 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 476 | master passed |
| +1 | javadoc | 136 | master passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 14 | Maven dependency ordering for patch |
| +1 | mvninstall | 246 | the patch passed |
| +1 | compile | 185 | the patch passed |
| +1 | javac | 185 | the patch passed |
| +1 | checkstyle | 189 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 263 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 532 | Patch does not cause any errors with Hadoop 2.7.4 or 3.0.0. |
| +1 | findbugs | 502 | the patch passed |
| +1 | javadoc | 140 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 203 | hbase-client in the patch passed. |
| -1 | unit | 16859 | hbase-server in the patch failed. |
| +1 | unit | 1315 | hbase-mapreduce in the patch passed. |
| +1 | unit | 309 | hbase-thrift in the patch passed. |
| +1 | unit | 1236 | hbase-backup in the patch passed. |
| +1 | unit | 466 | hbase-rest in the patch passed. |
| +1 | asflicense | 221 | The patch does not generate ASF License warnings. |
| | | 24549 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.quotas.TestSpaceQuotas |
|   | hadoop.hbase.master.procedure.TestSCPWithReplicasWithoutZKCoordinated |
|   | hadoop.hbase.replication.TestReplicationDisableInactivePeer |
|   | hadoop.hbase.client.TestSnapshotTemporaryDirectoryWithRegionReplicas |
|   | hadoop.hbase.client.TestFromClientSideWithCoprocessor |
|   | hadoop.hbase.master.procedure.TestSCPWithReplicas |
|   | hadoop.hbase.client.replication.TestReplicationAdminWithClusters |
|   | hadoop.hbase.client.TestFromClientSide3 |
|   | hadoop.hbase.client.TestAsyncTableAdminApi |
|   | hadoop.hbase.client.TestAdmin1 |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/173 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 4f28804ff981 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 184568cadd |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/3/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/3/testReport/ |
| Max. process+thread count | 5230 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server hbase-mapreduce hbase-thrift hbase-backup hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 63 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 17 new or modified test files. |
||| _ master Compile Tests _ |
| 0 | mvndep | 36 | Maven dependency ordering for branch |
| +1 | mvninstall | 339 | master passed |
| +1 | compile | 218 | master passed |
| +1 | checkstyle | 225 | master passed |
| +1 | shadedjars | 330 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 580 | master passed |
| +1 | javadoc | 172 | master passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 16 | Maven dependency ordering for patch |
| +1 | mvninstall | 292 | the patch passed |
| +1 | compile | 231 | the patch passed |
| +1 | javac | 231 | the patch passed |
| +1 | checkstyle | 223 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 326 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 667 | Patch does not cause any errors with Hadoop 2.7.4 or 3.0.0. |
| +1 | findbugs | 621 | the patch passed |
| +1 | javadoc | 182 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 204 | hbase-client in the patch passed. |
| +1 | unit | 13847 | hbase-server in the patch passed. |
| +1 | unit | 1247 | hbase-mapreduce in the patch passed. |
| +1 | unit | 275 | hbase-thrift in the patch passed. |
| +1 | unit | 996 | hbase-backup in the patch passed. |
| +1 | unit | 437 | hbase-rest in the patch passed. |
| +1 | asflicense | 190 | The patch does not generate ASF License warnings. |
| | | 21905 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/173 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux bfaa7bc756b7 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 3f40df8085 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/4/testReport/ |
| Max. process+thread count | 5234 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server hbase-mapreduce hbase-thrift hbase-backup hbase-rest U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-173/4/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**